index.html

```diff
@@ -94,6 +94,115 @@ <h2>
+          motivation for creating a new DID method instead of continuing to
+          use <code>did:web</code>.
+        </p>
+
+        <ul>
+          <li>Is crucial in "multi-tenancy" scenarios (either via subdomains
+          or subdirectories).</li>
+          <li>Allows for greater availability, archiving, and migration to
+          other domains or other DID methods.</li>
+        </ul>
+      </section>
+    </section>
+
+    <section>
+      <h2>
+        Relationship to Other DID Methods
+      </h2>
+
+      <p>
+        This DID method is an iteration over the original <code>did:web</code>
+        method, and exists in conversation with similar web-derived
+        microledger-based methods currently under development.
+      </p>
+      <ul>
+        <li><code>did:web</code> - No key rotation history; the DID Document
+          contains only the latest valid keys. Unsigned DID docs.</li>
+        <li><code>did:webs</code> - see the
+          <a href="https://trustoverip.github.io/tswg-did-method-webs-specification/">DID Method did:webs Specification</a>.
+          Key rotation events at a separate location (in KERI format).
+        </li>
+        <li><code>did:webplus</code> - see
+          <a href="https://github.com/LedgerDomain/did-webplus">https://github.com/LedgerDomain/did-webplus</a></li>
+        <li><code>did:plc</code> - key rotation events at a separate location</li>
+      </ul>
+
+      <div class="issue">Add text comparing did:webhash to did:webs and similar
+        methods.
+      </div>
+
+      <section>
+        <h3>Enhancements and Differences from <code>did:web</code></h3>
+
+        <p>The differences are nonbreaking and additive (see the Design
+          Constraints section).</p>
+
+        <ul>
+          <li>Introduces the <code>previous</code> term for backwards chaining,
+            as well as a corresponding algorithm for chain verification.</li>
+          <li>DID Documents must be signed.</li>
+          <li>Requires the use of the <code>controller</code> term (specifically,
+            DID document update and key rotation events must be signed by
+            the <code>controller</code> DID (specifically, by a key
+            authorized in its <code>assertionMethod</code> section).
+          </li>
+          <li>Requires the use of an entropic identifier as a path fragment.</li>
+          <li>Because there is always a path fragment, no <code>.well-known/</code>
+            directory mechanism is used for a DID at the "top" domain level.</li>
+          <li>Requires the use of Tombstones when a DID is deactivated.</li>
+          <li>Provides affordances for DID migration and archiving (for end-of-life
+            and migration from non-cooperative server scenarios).</li>
+          <li>Specifies optional but recommended Witness mechanism.</li>
+          <li>Specifies optional but recommended Cache Control property.</li>
```

Comment:
are these SPECIFIED or is a common interface specified for multiple possible witness and/or cache control mechanisms to be separately specified?

Reply:
The latter, I think.